src/ostorlab/agent/agent.py

```diff
@@ -18,6 +18,8 @@
 import uuid
 import json
 from typing import Dict, Any, Optional, Type, List
+import google.cloud.logging
+from google.oauth2 import service_account
```

Comment:
It should follow the imports convention.
```suggestion
from typing import Dict, Any, Optional, Type, List

import google.cloud.logging
from google.oauth2 import service_account
```